samples/snippets/storage_fileio_pandas.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+"""Sample that creates and consumes a GCS blob using pandas with file-like IO
+"""
+
+# [START storage_fileio_pandas]
+from google.cloud import storage
+
+
+def pandas_write_read(bucket_name, blob_name):
```

Comment:
We could consider breaking this out into a write and a read sample, and then running them one after the other in `__main__`. That way users copy/pasting can easily copy/paste the writes or reads as they need them, but the command-line sample still works the same way.

Reply:
Good idea, on it.